test/ERC6538Registry.t.sol

```diff
@@ -227,6 +254,20 @@ contract RegisterKeysOnBehalf is ERC6538RegistryTest {
     }
   }
 
+  function testFuzz_AManipulatedErc712SignatureIsUsedToRegisterTheSameStealthMetaAddress(
```

Comment:
Would update the test name to something like "Sets The Correct Stealth Meta Address Even If The Signature Is Manipulated Via Signature Malleability" or something similar.

Reply:
Updated [f9b7510](https://github.com/ScopeLift/stealth-address-erc-contracts/pull/9/commits/f9b7510ab6be32699b497f507e59836b6b2d159c)